x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -143,12 +155,13 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 			)
 		}
 
-		isValidPairing, allowedCU, servicersToPair, projectID, err := k.Keeper.ValidatePairingForClient(
+		isValidPairing, allowedCU, servicersToPair, err := k.Keeper.ValidatePairingForClient(
 			ctx,
 			relay.SpecId,
 			clientAddr,
```

Comment:
if we give the project than we dont need the clientAddr, it is the project name
remove it

Reply:
it's not true since clientAddr can come from the badge. So it needs to be kept

Reply:
my bad. see fix here: https://github.com/lavanet/lava/commit/1270ca49fda72e555e5fac2d52a51187b63c96ba